app/src/main/res/layout/layout_tokens_item_balance.xml

```diff
@@ -53,4 +57,13 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:text="3342434.23545 ETH" />
 
+    <View
```

Comment:
shouldn'T it be the first view do be declared. Now the ripple is drawing over all the views, normally it is drawn behind all the views (directly on the background)